Fix #13937 make ParamTable always render so Chrome search can detect text

paritoshdey-dev opened the pull request:
Here's the screenshot of the local test:

<img width="1911" height="959" alt="image" src="https://github.com/user-attachments/assets/831bf013-93cb-4caa-b992-f53544aaff09" />

paritoshdey-dev committed:
Fix: make ParamTable always render so Chrome search can detect text (#13937)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/meteor/meteor?pullRequest=13941) <br/>All committers have signed the CLA.

Grubba27 commented:
Not sure if I liked this.... It makes the docs look too bloated. Maybe we can think of something else. Hijacking/listening to what is being searched idk.

@italojs https://deploy-preview-13941.docs-online.meteor.com/api/collections.html#Mongo-Collection

When hitting the cmd+k search, it could open the args tables; 

One thing that could help the crawler is to make it open by default to crawlers

wreiske commented:
It's important to link the original issue with documentation on why the current experience ~sucks~ is not great. https://github.com/meteor/meteor/issues/13937

italojs commented:
It's bloated but is better than an unfindable doc 
Unfortunately, nobody reads the docs with a high level of attention, usually we just do a few ctrl+f or ask for AI in this new AI era 🤷🏻‍♂️
Another thing is SEO, as we can see here, collapsed docs can't be readable by google's engine
I just see benefits here

sanki92 committed:
docs: clarify forbidClientAccountCreation requires client+server config (#13961)

- Add clear warning that forbidClientAccountCreation must be set on both client and server
- Include practical example showing shared config file approach
- Fixes issue where UI still shows 'Create account' link when only set on server

Fixes #13922

Co-authored-by: Italo José <italo.i@live.com>

StorytellerCZ committed:
Add docs for OAuth configurations (#13946)

Also includes auto generated stuff

Co-authored-by: Italo José <italo.i@live.com>

vparpoil committed:
Docs: bring Production > Security content from the Guide to v3 docs (#13934)

* feat: add security from meteor guide to v3-docs tutorial folder

* fix: use markdown tags for titles

* fix: rework on methods

* fix: finalize changes of urls and adaptations to jam:method

---------

Co-authored-by: Italo José <italo.i@live.com>

tajagn01 committed:
docs: add documentation for loginWithToken in accounts.md (#13839)

* Duplicate forEachAsync definition removed.

* going back

* docs: add loginWithToken to accounts.md

* repush the brench

* Restore accounnt-ui documentation line and keel loginWithToken docs

* Fix formatting in accounts.md documentation

* Fix formatting issue in accounts.md

---------

Co-authored-by: Italo José <italo.i@live.com>

italojs committed:
Merge branch 'devel' into fix/docs-paramtable-search

Grubba27 commented:
I still think that there is too much information out of the box with this, but it can be fine....

As a future improvement, we could have it stored in local storage, with the user's preference on whether the table should be shown by default or not.

For crawlers(AI, Regular Crawlers, and Algolia), it could be true by default idk
